elifesciences/eLife-JATS-schematron
# Improvements

- Upgrade `surname-test-4` to error
- Allow brackets in `given-names-test-5`
- Add `journal-conference-ref-check`, closes #106
- Add query for producing local validaiton report `validation-report.xq`
